CustomApps/new-releases/index.js

```diff
@@ -269,8 +269,15 @@ class Grid extends react.Component {
 }
 
 async function getArtistList() {
-	const base = await Spicetify.Platform.LibraryAPI.getArtists();
-	const artists = await Spicetify.Platform.LibraryAPI.getArtists({ limit: base.totalLength });
+	const config = {
+		filters: ["1"],
+		sortOrder: ["0"],
+		textFilter: "",
+		offset: 0,
+		limit: 0,
+	};
+	config.limit = (await Spicetify.Platform.LibraryAPI.getContents(config)).totalLength;
```

Comment:
```suggestion
		limit: (await Spicetify.Platform.LibraryAPI.getContents(config))?.totalLength ?? 0,
	};
```

Reply:
Cleaner for sure!
Not sure why I didn't do that to begin with...